### PR TITLE
platformio 4.3.0

### PR DESCRIPTION
**Pull request by ivankravets**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/platformio.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Platformio < Formula
 
   desc "An open-source ecosystem for embedded development"
   homepage "https://platformio.org/"
-  url "https://files.pythonhosted.org/packages/f4/c8/8e472c22602ec6cdc6f93e35357ff5f3dff3434da6fc76cd3cd57c7eefb6/platformio-4.2.1.tar.gz"
-  sha256 "310fa8b624cf938ef95284ae3a40a9caa36aafef32ac30ad588b43df516945c3"
+  url "https://files.pythonhosted.org/packages/0d/cd/6e7a07fa7bf9c52f7d69572f17fd58feee74013de03519698b44b4735d85/platformio-4.3.0.tar.gz"
+  sha256 "27a34001a622fdf6478562f08d15d5624b11bc702343435802a8dfe6526defcd"
   revision 1
```
Comment (dawidd6) [inline Formula/platformio.rb]: Remove this line please.

Comment (ivankravets) [reply Formula/platformio.rb]: Every time I remove this line and Homebrew bot adds it again. Why?

Comment (dawidd6) [reply Formula/platformio.rb]: It's not added by test-bot.

See: https://github.com/Homebrew/homebrew-core/commit/2be0f45aa54dd5dd45155d91b80221ffaf52d8ed

Comment (ivankravets) [reply Formula/platformio.rb]: Thanks! What does mean that commit by @Bo98?

**Pull request closed without merging**